setup.py

```diff
@@ -90,8 +90,8 @@ def get_package_data(package):
     ],
     install_requires=[
         'inflection>=0.3.0',
-        'djangorestframework>=3.10',
-        'django>=1.11',
+        'djangorestframework>=3.10,<3.12',
+        'django>=1.11,<3.1',
```

Comment:
why not drop django below 2.2 and add support for 3.1?

Reply:
We only drop support for Django 1.11 once DRF 3.12 is released as earlier versions still support 1.11. And Django 3.1 can not yet be supported as I have commented [above](https://github.com/django-json-api/django-rest-framework-json-api/pull/808#issuecomment-675367931)